core/templates/core/post-edit.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<div class="container mt-5">
+    <h1 class="mb-4">Edit Post</h1>
+
+    {% if form.errors %}
+    <div class="alert alert-danger" role="alert">
+        <ul class="mb-0">
+            {% for error in form.non_field_errors %}
+                <li>{{ error }}</li>
+            {% endfor %}
+        </ul>
+    </div>
+    {% endif %}
+
+    <form method="post">
+        {% csrf_token %}
+
+        <div class="mb-3">
+            <label for="{{ form.title.id_for_label }}" class="form-label">{{ form.title.label }}</label>
```

Comment:
For the record: this is just a remark, not a change request. This code is good. 

Have you maybe checked other forms?
The way you do it here is good, however going field over field with copy/paste is bad practice and usually a busywork.
We typically use https://github.com/pymasterspl/reddit/blob/08e1ae4566f1f5c5cfb6aed5c09f0db3a0a41697/core/templates/core/post-detail.html#L53 - form|crispy and it does the job. 

If you really want to go field over field, and interesting approach is to loop over form.fields, to minimize code repetition.